client/stylesheets/_modal.scss

```diff
@@ -1071,11 +1083,15 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 }
 
 .modFeedbackPageTwo2 ul li textarea {
-	width: 66%;
+	width: 80%;
 	height: 100px;
+	border-radius: 4px;
+	padding: 10px 12px;
+	border: 1px solid #d3d3d3;
 }
 
-.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"] {
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"], 
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `modFeedbackInitial` should be written in lowercase with hyphens<br>Line contains trailing whitespace